VideoLocker/res/values/styles.xml

```diff
@@ -42,6 +42,7 @@
         <item name="android:progressBarStyleInverse">@style/edX.Widget.ProgressBar.Inverse</item>
         <item name="android:progressBarStyleSmallInverse">@style/edX.Widget.ProgressBar.Small.Inverse</item>
         <item name="android:progressBarStyleLargeInverse">@style/edX.Widget.ProgressBar.Large.Inverse</item>
+        <item name="alertDialogTheme">@style/AppTheme.Dialog.Alert</item>
```

Comment:
Hmm, true but I would rather we just switched those to use the AppCompat dialog.